### PR TITLE
Update docs for CEL Authorization rules

### PR DESCRIPTION
**Pull request by dimas-b**
* Update available CEL variables for all check types

* Fix links

### CODE REVIEW

File: site/in-dev/authorization.md [hunk 1]
```diff
@@ -75,32 +75,46 @@ Rule definitions are of the form `nessie.server.authorization.rules.<ruleId>=<ru
...
+Available variables within the `<rule_expression>` are: **'op'** / **'role'** / **'roles'** / **'ref'** / **'path'** / **'contentType'**.
 
-* The **'op'** variable in the `<rule_expression>` refers to the type of operation can be any of the following.
+The **'op'** variable in the `<rule_expression>` refers to the type of operation can be any of the following.
   See [BatchAccessChecker](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/BatchAccessChecker.java)
-  and [Check](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/Check.java) types.
-  * Arguments to the CEL script for the following check-types:
+  and [Check](https://github.com/projectnessie/nessie/blob/main/servers/services/src/main/java/org/projectnessie/services/authz/Check.java) types for details.
+
+* Checks for Reference operations:
```
Comment (snazy) [inline site/in-dev/authorization.md:84]:
```suggestion
#### Checks for Reference operations

```

File: site/in-dev/authorization.md [hunk 2]
```diff
@@ -75,32 +75,46 @@ Rule definitions are of the form `nessie.server.authorization.rules.<ruleId>=<ru
...
+    * **'ref'** refers to a string representing a branch/tag name or `DETATCHED` for direct access to a commit id.
+  * **'op'** types:
+    * `VIEW_REFERENCE`
+    * `CREATE_REFERENCE`
+    * `DELETE_REFERENCE`
+    * `ASSIGN_REFERENCE_TO_HASH`
+    * `DELETE_REFERENCE`
+    * `READ_ENTRIES`
+    * `LIST_COMMIT_LOG`
+* Checks for Content operations:
```
Comment (snazy) [inline site/in-dev/authorization.md:97]:
```suggestion
#### Checks for Content operations

```

File: site/in-dev/authorization.md [hunk 3]
```diff
@@ -75,32 +75,46 @@ Rule definitions are of the form `nessie.server.authorization.rules.<ruleId>=<ru
...
-  * Arguments to the CEL script for the repository config check-types:
+    * **'path'** refers to the [content key](https://github.com/projectnessie/nessie/blob/main/api/model/src/main/java/org/projectnessie/model/ContentKey.java) for the contents of an object and can be any string
+    * **'contentType'** refers to a (possibly empty) string representing object's [content type](https://github.com/projectnessie/nessie/blob/main/api/model/src/main/java/org/projectnessie/model/Content.java).
+  * **'op'** types:
+    * `READ_CONTENT_KEY`
+    * `READ_ENTITY_VALUE`
+    * `CREATE_ENTITY`
+    * `UPDATE_ENTITY`
+    * `DELETE_ENTITY`
+* Checks for Repository Config operations
```
Comment (snazy) [inline site/in-dev/authorization.md:110]:
```suggestion
#### Checks for Repository Config operations

```

File: site/in-dev/authorization.md (hunk 1 above)
Comment (snazy) [reply site/in-dev/authorization.md:84]: And unindent the following lists (same below)

**Review by snazy — COMMENTED** [review]
Thanks for tackling this!

Just style proposal comments

File: site/in-dev/authorization.md (hunk 1 above)
Comment (dimas-b) [reply site/in-dev/authorization.md]: Thatnks for the tip.... In the end I ended up re-formatting this text quite a bit... I hope for the better :)

**Review by snazy — APPROVED** [review]

**Pull request merged**